<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1366</width>
    <height>774</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTableWidget" name="splineTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>190</y>
      <width>461</width>
      <height>291</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="startButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>110</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Старт</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numMainGrid">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>110</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>100</string>
    </property>
   </widget>
   <widget class="QComboBox" name="funcPar">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>141</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="funcParTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>47</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Функция</string>
    </property>
   </widget>
   <widget class="QLabel" name="rangeTxt">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>251</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>На отрезке от a =                   до b = </string>
    </property>
   </widget>
   <widget class="QTableWidget" name="errorTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>1061</width>
      <height>241</height>
     </rect>
    </property>
   </widget>
   <widget class="QTabWidget" name="tabgr">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>10</y>
      <width>881</width>
      <height>471</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="tabgr0">
     <attribute name="title">
      <string>Сплайн и функция</string>
     </attribute>
     <widget class="QCustomPlot" name="graph0" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabgr1">
     <attribute name="title">
      <string>Первые производные сплайна и функции</string>
     </attribute>
     <widget class="QCustomPlot" name="graph1" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tabgr2">
     <attribute name="title">
      <string>Вторые производные сплайна и функции</string>
     </attribute>
     <widget class="QCustomPlot" name="graph2" native="true">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>871</width>
        <height>441</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="a_rangePar">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>-1</string>
    </property>
   </widget>
   <widget class="QLabel" name="b_rangePar">
    <property name="geometry">
     <rect>
      <x>430</x>
      <y>20</y>
      <width>41</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>1</string>
    </property>
   </widget>
   <widget class="QLabel" name="brdTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>211</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Интерполируется кубическим сплайном &lt;/p&gt;&lt;p&gt;с граничными условиями в виде:&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QComboBox" name="brdCondPar">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>60</y>
      <width>111</width>
      <height>41</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>Естественные ГУ</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Совпадение с f ''</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Вручную</string>
     </property>
    </item>
   </widget>
   <widget class="QLineEdit" name="a_brdPar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>60</y>
      <width>71</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="b_brdPar">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>80</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QLabel" name="a_brdTxt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>S''(a) = </string>
    </property>
   </widget>
   <widget class="QLabel" name="b_brdTxt">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>80</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>S''(b) = </string>
    </property>
   </widget>
   <widget class="QLabel" name="numMainTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>211</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>На сетке с количеством разбиений n = </string>
    </property>
   </widget>
   <widget class="QLabel" name="numAuxTxt">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Численная оценка сходства функции и сплайна проводится&lt;/p&gt;&lt;p&gt;на сетке c количеством разбиений N = &lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="numAuxGrid">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>161</y>
      <width>61</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>400</string>
    </property>
   </widget>
   <widget class="QLabel" name="max0Txt">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>540</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>max |f(x)-S(x)|    =                           в x = </string>
    </property>
   </widget>
   <widget class="QLabel" name="max1Txt">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>590</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>max |f'(x)-S'(x)|   =                           в x = </string>
    </property>
   </widget>
   <widget class="QLabel" name="max2Txt">
    <property name="geometry">
     <rect>
      <x>1080</x>
      <y>640</y>
      <width>211</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>max |f ''(x)-S''(x)| =                           в x = </string>
    </property>
   </widget>
   <widget class="QLabel" name="max0Val">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>540</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max0X">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>540</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max1Val">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>590</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max1X">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>590</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max2Val">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>640</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="max2X">
    <property name="geometry">
     <rect>
      <x>1290</x>
      <y>640</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="clearButton">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>152</y>
      <width>111</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Очистить</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1366</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Меню</string>
    </property>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
